llvm/lib/Target/AArch64/GISel/AArch64LegalizerInfo.cpp

```diff
@@ -646,6 +646,7 @@ AArch64LegalizerInfo::AArch64LegalizerInfo(const AArch64Subtarget &ST)
 
   // FP conversions
   getActionDefinitionsBuilder(G_FPTRUNC)
+      .libcallIf(typeIs(1, s128))
```

Comment:
I would prefer `legalFor` as the first line.

Reply:
Could it be a list: `.libcall({{s16, s128}, {s32, s128}, {s64, s128}})`.

Reply:
Yes, that sounds better. It is `libcallFor`